README.md

```diff
@@ -1,4 +1,681 @@
+  - 404: Topic specified does not exist.
+  - 422: Semantically invalid request.
+
+#### Example
+
+```golang
+	// Construct an instance of the UpdateTopicOptions model
+	updateTopicOptionsModel := new(ibmeventstreamsadminrestapiv1.UpdateTopicOptions)
+	updateTopicOptionsModel.TopicName = core.StringPtr("test-topic")
+	updateTopicOptionsModel.NewTotalPartitionCount = core.Int64Ptr(int64(6))
+	updateTopicOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+
+	// Invoke operation with valid options model (positive test)
+	response, operationErr := serviceAPI.UpdateTopic(updateTopicOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Updating Topic: %s\n", operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusAccepted {
+		return fmt.Errorf("Error Updating Topics: status %d\n", response.StatusCode)
+	}
+
+	fmt.Printf("\tname: %s updated\n", *updateTopicOptionsModel.TopicName)
+
+```
+
+### List current mirroring topic selection
+
+Mirroring user controls are available on the target cluster in a mirroring environment.
+
+To get the current topic selection, issue an GET request to /admin/mirroring/topic-selection
+
+
+Expected status codes
+    - 200: Retrieved topic selection successfully in following format:
+	{
+	"includes": [
+		"^prefix1_.*",
+		"^prefix2_.*"
+	]
+	}
+    - 403: Unauthorized to use mirroring user controls.
+    - 404: Mirroring not enabled. The mirroring user control apis are only available on a target in a - pair of clusters with mirroring enabled between them.
+    - 503: An error occurred handling the request.
+
+#### Example
+
+```golang
+	// Construct an instance of the GetMirroringTopicSelectionOptions model
+	getMirroringTopicSelectionOptionsModel := new(ibmeventstreamsadminrestapiv1.GetMirroringTopicSelectionOptions)
+	getMirroringTopicSelectionOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+
+	// Enable retries.
+	serviceAPI.EnableRetries(0, 0)
+
+	// Call GetMirroringTopicSelection.
+	result, response, operationErr := serviceAPI.GetMirroringTopicSelection(getMirroringTopicSelectionOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Listing Mirroring Topics: %s\n", operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusAccepted {
+		return fmt.Errorf("Error Listing Mirroring Topics: status %d\n", response.StatusCode)
+	}
+
+	// Loop and print mirroring topics.
+	for _, topicName := range result.Includes {
+		fmt.Printf("\tname: %s\n", topicName)
+	}
+```
+
+### Replace selection of topics which are mirrored
+
+Replace mirroring topic selection
+
+Mirroring user controls are available on the target cluster in a mirroring environment.
+
+To replace the current topic selection, issue a POST request to /admin/mirroring/topic-selection
+
+Expected status codes
+
+    200: Replaced topic selection successfully. The new selection is returned in following format:
```

Comment:
This does not render correctly, see https://github.com/IBM/eventstreams-go-sdk/blob/980eb5aa082f35ca4929fbe655d6806cb93d54eb/README.md#replace-selection-of-topics-which-are-mirrored

Reply:
fixed that in template.